src/solc_0.5/Estate.sol

```diff
@@ -27,20 +33,20 @@ contract Estate is EstateBaseToken {
     }
 
     // solium-disable-next-line security/no-assign-params
-    function uint2str(uint256 _i) internal pure returns (string memory) {
+    function uint2str(uint _i) internal pure returns (string memory) {
```

Comment:
I'm wondering why only some instances of the uint256 type are using the uint alias...

Reply:
I think what is happening is that old_src must have been through some formatting at some point
and now with this PR I am bring the original old_src from an earlier version